build/automation/var/platform-texas/v1/account-live-k8s-nonprod.mk

```diff
@@ -1,5 +1,8 @@
 AWS_ACCOUNT_ID := $(AWS_ACCOUNT_ID_NONPROD)
 AWS_ACCOUNT_NAME := nonprod
+AWS_CERTIFICATE = c0718115-4e22-4f48-a4aa-8c16ea86c5e6
```

Comment:
@stefaniuk : are we happy to have cert-ids and similar here? Or would we prefer to inject?

Reply:
@stefaniuk confirmed OK with @jope1 👍